src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1602,31 +1596,39 @@ private String getContainerCommand() {
         }
 
         // mount potential directories containing .war.xml from devc specific folder - override /config/apps and /config/dropins
-        command.append(" -v " + buildDirectory + "/" + DEVC_HIDDEN_FOLDER + "/apps:/config/apps");
-        command.append(" -v " + buildDirectory + "/" + DEVC_HIDDEN_FOLDER + "/dropins:/config/dropins");
+        File tempApps = new File(buildDirectory, DEVC_HIDDEN_FOLDER + "/apps");
+        File tempDropins = new File(buildDirectory, DEVC_HIDDEN_FOLDER + "/dropins");
+        if (!tempApps.exists()) {tempApps.mkdirs();}
```

Comment:
So the code I added for this was in method `startContainer()`. Should we be doing this here and there? If it legitimately needs to be done in both places, maybe we should pull out into a separate method to check for the existence of all three directories and create them if needed (apps, dropins, logs).

Reply:
Seemed to work fine without these additions when I did some testing. I'll remove the checks here.